umap/static/umap/vendors/markercluster/MarkerCluster.Default.css

```diff
@@ -48,7 +48,7 @@
 .marker-cluster div {
 	width: 30px;
 	height: 30px;
-	margin-left: 5px;
```

Comment:
This is not possible to modify CSS from the `vendors` folder, is that critical to left it as is?

Reply:
This fixes cluster icon issue described in #2038

Reply:
I get it but the `vendors` folder contains files we copy/overwrite from `node_modules` installations. If you want to override that CSS rule, you need to do it within one of our under-control CSS file, I would say `map.css`.

(`vendors` are only commited to ease the deployment/release process.)